.gitlab-ci.yml

```
@@ -0,0 +1,50 @@
+variables:
+  CI_DOCKER_IMAGE: registry.ddbuild.io/images/docker@sha256:23643e6da71fc2551046a1928c071e6db5770f050a0f5a1e613168d2c65fb8bc # Docker 20.10.13
+  DOCKER_CONTEXT: "."
+  DOCKER_BUILD_ARGS: ""
+
+.build-docker-image: &build-docker-image
+  stage: build
+  image: $CI_DOCKER_IMAGE
+  tags: ["runner:docker"]
+  rules:
+    # Automatically run the pipeline for all pushed tags
+    - if: $CI_COMMIT_TAG
+  script:
+    - set -x
+    - builder=$(docker buildx create --use)
+    - trap "docker buildx rm $builder" EXIT
+    # Construct valid --build-args arguments from the DOCKER_BUILD_ARGS variable
+    - BUILD_ARGS=""; IFS=$'\n'; for arg in $DOCKER_BUILD_ARGS; do BUILD_ARGS+=" $(echo "--build-arg $arg")"; done; IFS=$' ';
+    - IMAGE_TAG="registry.ddbuild.io/$IMAGE_NAME:$CI_COMMIT_TAG"
+    - docker buildx build --platform linux/amd64,linux/arm64 --tag $IMAGE_TAG --file $DOCKERFILE_PATH $BUILD_ARGS --label CILIUM_VERSION=$(cat VERSION) --label target=prod --push $DOCKER_CONTEXT
+
+build-docker-image-operator:
+  <<: *build-docker-image
+  variables:
+    IMAGE_NAME: cilium-operator
+    DOCKERFILE_PATH: images/operator/Dockerfile
+    DOCKER_BUILD_ARGS: |
+      BASE_IMAGE=registry.ddbuild.io/images/base/gbi-distroless:release@sha256:be05d4c795fa14da8c21ef40f0f4dd257095adf204a44e2b24676a92c03ea52d
```

```suggestion
      BASE_IMAGE=registry.ddbuild.io/images/base/gbi-distroless:release@sha256:eb29f16c4a6557486bad50b6ba269670d9db47697f62c07aa3673e37600993c8
```

Let's use the latest sha